libkbfs/mdserver_memory.go

```diff
@@ -610,7 +658,10 @@ func (md *MDServerMemory) Put(ctx context.Context, rmds *RootMetadataSigned,
 		return kbfsmd.ServerError{Err: err}
 	}
 
-	block := mdBlockMem{encodedMd, md.config.Clock().Now(), rmds.MD.Version()}
+	// Pretend the timestamp went over RPC, so we get the same
+	// resolution level as a real server.
+	t := keybase1.FromTime(keybase1.ToTime(md.config.Clock().Now()))
```

Comment:
nice

Reply:
We should probably do it [here](https://github.com/keybase/kbfs/blob/ddf361693e24e7a1c011011ee58d8a331f76febd/libkbfs/mdserver_tlf_storage.go#L187) too?